src/App.tsx

```diff
@@ -1,18 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(3);
+
   return (
     <div className="container">
       <h1>Items with Pagination</h1>
 
       <p className="lead" data-cy="info">
-        Page 1 (items 1 - 5 of 42)
+        {`Page ${currentPage} (items ${currentPage * perPage - perPage + 1} - ${currentPage * perPage} of ${items.length})`}
```

Comment:
```suggestion
        {`Page ${currentPage} (items ${currentPage * perPage - perPage + 1} - ${currentPage * perPage} of ${items.length})`}
```
I think it needs to be simplified